### PR TITLE
Add tx-pause to Manta Runtime

### PR DESCRIPTION
**Pull request by ferrell-code**
## Description

Adds tx-pause pallet to manta runtime

Fixes non-existent pallet names (`CouncilCollective` -> `CouncilMembership`) in `NonPausablePallets` and add integration test for this case

merge after #1087 

---

Before we can approve this PR for merge, please make sure that **all** the following items have been checked off:
- [x] Connected to an issue with discussion and accepted design using zenhub "Connect issue" button below
- [x] Added **one** label out of the `L-` group to this PR
- [x] Added **one or more** labels from the `A-` and `C-` groups to this PR
- [x] Explicitly labelled `A-calamari` and/or `A-manta` if your changes are meant for/impact either of these (CI depends on it)
- [x] Re-reviewed `Files changed` in the Github PR explorer.

Situational Notes:
- If adding functionality, write unit tests!
- If importing a new pallet, choose a proper module index for it, and allow it in `BaseFilter`. Ensure **every** extrinsic works from front-end. If there's corresponding tool, ensure both work for each other.
- If needed, update our Javascript/Typescript APIs. These APIs are officially used by exchanges or community developers.
- If modifying existing runtime storage items, make sure to implement storage migrations for the runtime and test them with `try-runtime`. This includes migrations inherited from upstream changes, and you can search the diffs for modifications of `#[pallet::storage]` items to check for any.

### CODE REVIEW

**Review by ghzlatarev — DISMISSED** [review]

File: runtime/integration-tests/src/integrations_mock/test_common.rs [hunk 1]
```diff
@@ -696,6 +700,164 @@ fn concrete_fungible_ledger_transfers_work() {
         });
 }
 
+fn pause_transaction_storage_event_works(pause: bool) {
```
Comment (Garandor) [inline runtime/integration-tests/src/integrations_mock/test_common.rs]: pls make this a `mod tx_pause_tests` as parastaking above. That makes if fold away more easily

**Review by Garandor — CHANGES_REQUESTED** [review]
test structure change, otherwise LGTM

File: runtime/integration-tests/src/integrations_mock/test_common.rs (hunk 1 above)
Comment (ferrell-code) [reply runtime/integration-tests/src/integrations_mock/test_common.rs]: Sure thing

we probably should separate them into different files as more lines are added. That would be better done in different PR

**Review by Garandor — APPROVED** [review]

**Review by ghzlatarev — APPROVED** [review]

File: runtime/manta/src/weights/pallet_tx_pause.rs [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+//!
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
+//! DATE: 2023-04-20, STEPS: `50`, REPEAT: 20, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! EXECUTION: Some(Wasm), WASM-EXECUTION: Compiled, CHAIN: Some("calamari-dev"), DB CACHE: 1024
+
+// Executed Command:
+// ./target/production/manta
+// benchmark
+// pallet
+// --chain=calamari-dev
```
Comment (Dengjianping) [inline runtime/manta/src/weights/pallet_tx_pause.rs:27]: Am I wrong about the command to generate weight file for manta runtime? It should be something like `manta-dev`.
Or will you generate it by a release PR?

Comment (ferrell-code) [reply runtime/manta/src/weights/pallet_tx_pause.rs:27]: Yeah I copied the weights from calamari...

This should be updated in release

Comment (ghzlatarev) [reply runtime/manta/src/weights/pallet_tx_pause.rs:27]: yeah we'll update it on the release PR

**Pull request merged**